src/lib/controllers/ui-controller.js

```diff
@@ -576,7 +576,8 @@ export default class UIController {
         },
 
         setWordLists (state, wordLists) {
-          state.hasWordListsData = Boolean(wordLists.find(wordList => !wordList.isEmpty))
+          let checkWordLists = Array.isArray(wordLists) ? wordLists : Object.values(wordLists)
```

Comment:
why do we get different structures? Should we fix that instead?

Reply:
Answered in PR - https://github.com/alpheios-project/components/pull/438#discussion_r276079979